test/Templates.Test/Helpers/AddFirewallExclusion.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Templates.Test.Helpers
+{
+    internal class AddFirewallExclusion : IDisposable
+    {
+        private bool _disposedValue = false;
+        private readonly string _exclusionPath;
+
+        public static IDisposable Create(string exclusionPath)
```

Comment:
Based on the usage below, it looks like this class could just have a regular `public` constructor instead of a `static` factory function. Not a big deal - I'm not fixated on whether this is changed or not.